library-valcon-internship/src/components/BookCard/BookCard.tsx

```diff
@@ -0,0 +1,61 @@
+import placeholderBook from '../../assets/icons/placeholder-book.png'
+import Book from '../../models/Book'
+import './BookCard.css'
+
+interface BookCardProps {
+  book: Book
+}
+
+const BookCard = ({ book }: BookCardProps) => {
+  let descriptionLen = 0
+  if (book.Description) {
+    descriptionLen = book.Description.length
+    if (descriptionLen > 20) descriptionLen = 20
+  }
+  const publishDate = new Intl.DateTimeFormat('sr-RS').format(new Date(book.PublishDate))
+  let author = ''
+  if (book.Authors.length > 0) {
+    if (book.Authors.length > 1) {
+      author = `${book.Authors[0].FirstName} ${book.Authors[0].LastName}...`
+    } else {
+      author = `${book.Authors[0].FirstName} ${book.Authors[0].LastName}`
+    }
+  } else {
+    author = 'Unknown'
+  }
```

Comment:
You can always get all authors and then control the display with: https://caniuse.com/?search=line-clamp